addons/web/static/src/views/fields/many2many_tags_avatar/many2many_tags_avatar_field.js

```diff
@@ -27,13 +31,17 @@ export class Many2ManyTagsAvatarField extends Many2ManyTagsField {
 export const many2ManyTagsAvatarField = {
     ...many2ManyTagsField,
     component: Many2ManyTagsAvatarField,
+    extractProps: (fieldInfo) => ({
+        ...many2ManyTagsField.extractProps(fieldInfo),
+        viewType: fieldInfo.viewType,
```

Comment:
```suggestion
        maxItems: fieldInfo.viewType === "list" ? 5 : 3,
```
Or maybe only set it when in list, and have a default props which is 3